tedana/workflows/tedana.py

```diff
@@ -150,6 +150,7 @@ def _get_parser():
             "in which case the specificed number of components will be "
             "selected."
         ),
+        choices=["mdl", "kic", "aic"],
```

Comment:
Believe this is re-introducing the error that was rectified in https://github.com/ME-ICA/tedana/pull/950.

Reply:
Correct. It is removed.